Yoast/wordpress-seo
# Improves stemming of some common irregular nouns in Slovak

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* [Yoast SEO Premium] Improves word form recognition of some irregular nouns in Slovak.
* [yoastseo] Adds an exception check for Slovak words that are on the exception lists of words for which not all forms are stemmed correctly.

## Relevant technical choices:

* The PR should be tested together with the[ premium-configuration PR](https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/233).

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Test in Premium
* Set your site to Slovak
* Open a post and enter a text of at least 100 words.
* Add `pes` as your keyphrase.
* Add the word `psa` somewhere in the text. 
* Make sure that the keyword density assessment says that 1 instance of the keyphrase is found in the text, and that the eye marker highlights the word `psa`.
* Change your keyphrase to `zvierať`.
* Enter the word `zvier` somewhere in the text.
* Make sure that the keyword density assessment says that 1 instance of the keyphrase is found in the text, and that the eye marker highlights the word `zvier`.


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-874
